templates/private/dashboard.html

```diff
@@ -63,7 +63,7 @@ <h3>Key Creation</h3>
         {# Key count input #}
         <div class="form-group">
             <label for="key_count">Generate keys</label>
-            <input type="number" class="form-control" name="key_count" placeholder="Enter number of keys...">
+            <input type="number" class="form-control" name="key_count" placeholder="Enter number of keys..." required min="1">
```

Comment:
The [default `step` value is 1](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-step) which therefore also makes sure that only integer values can be provided.